src/screens/InboundReceiveDetail/index.tsx

```diff
@@ -246,6 +248,11 @@ const InboundReceiveDetail = () => {
           onChange={onChangeComment}
         />
       </View>
+      <Radio
+        title={"Cancel all remaining"}
+        setValue={setCancelRemaining}
+        disabled={Number(state.quantityToReceive) === Number(shipmentItem.quantityRemaining)}
```

Comment:
I think disabled should be set to true when the `Number(shipmentItem.quantityRemaining) <= 0`

Reply:
The checkbox should be enabled only when receive partial qty, it should be disabled when receive all qty. I have given condition based on the requirements